src/scripts/api.ts

```diff
@@ -30,7 +30,19 @@ interface QueuePromptRequestBody {
   number?: number
 }
 
+export interface LogEntry {
+  t: string
+  m: string
+}
+
+export interface TerminalSize {
+  cols: number
+  rows: number
+}
+
 class ComfyApi extends EventTarget {
+  #clientIdBlock: Promise<void>
```

Comment:
Why this `clientId` initialization is necessary here?

If we need to wait for the initial status message, we can explicitly do that at App level in workspaceStateStore or executionStore, and wait for that before calling `subscribeLogs`.